Proiect-VS/assets/css/login.less

```diff
@@ -0,0 +1,116 @@
+body {
+    margin: auto;
+}
+
+.skin_body {
+    font-family: Arial;
+}
+
+.banner {
+    position: relative;
+    opacity: 0.6;
+} 
+
+.login {
+    position: absolute;
+    top: 80px;
+    left: 300px;
+    width: 300px;
+    height: 250px;
+    border: none;
+    border-radius: 3px;
+}
+
+.skin_login {
+     background-color: white;
+     box-shadow: 0px 2px 2px #333435;   
+}
+
+.login_header {    
+    position: absolute; 
+    width:100%;
+    border-radius: 3px;
+    padding-top: 25px;
+    text-align: center;
+    vertical-align: middle;
+    line-height: 20px;
+    height: 40px;
+}
+
+.skin_login_header {
+    background-color: #419fe2; 
+    color: white;
+    font-size: 20px;
+}
+
+.login_input {
+    border: none;
+    border-bottom: 0.5px solid grey;
+    width: 80%;
+    margin-left: 30px;
+    margin-right: 10px;
+}
+
+.login_submit {    
+    border: none;    
+    padding: 5px 30px;
+    width: 35%;
+    margin-left: 95px;
+    border-radius: 3px;    
+}
+
+.skin_login_submit {
+    background-color: #419fe2;
+    color: white;
```

Comment:
Check if the font/color combination isn't used anywhere else in which case extract it into a standalone class for re-usability